HoloStorageAccessor/internal/holostorageaccessor/api_default_test.go

```diff
@@ -0,0 +1,1470 @@
+package holostorageaccessor
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+// This creates a test server that mocks the FHIR server responses
```

Comment:
Maybe a separate file?

Reply:
shifted it to a separate file `mock_server_test.go`